src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.BSD.cs

```diff
@@ -15,6 +15,7 @@ public partial class Process
         /// </summary>
         [UnsupportedOSPlatform("ios")]
         [UnsupportedOSPlatform("tvos")]
```

Comment:
This file since this file is not used on iOS and tvOS. Should the `UnsupportedOSPlatform` attributes be deleted here to match what we are doing in other places?

Reply:
I believe they should be deleted because they appear in `Process.iOS.cs`. I'll eventually get rid of it.

Reply:
Do you plan to clean this up in this PR?

Reply:
The point of earlier PR https://github.com/dotnet/runtime/pull/61670 was to make these attributes out of sync across the partial slices. And this PR has number of cases where the attributes are out of sync between slices, for example: https://github.com/dotnet/runtime/pull/61507/files#diff-99900ed5226bb1ffe4f6e912273d0782d17db3d8811d470def7e4988c8410d70R73

It would be nice to have some consistency here: Either have the attributes always in sync across the partial slices, or always have the attributes minimal in each partial slice. Thinking about it - having the attributes always in sync across the partial slices is easier to get right, even though it will result into a bit more duplication.

Reply:
I modified attributes usage in this PR in an earlier commit today. These particular attributes have to stay in place as @akoeplinger says but I was able to remove several other unnecessary attributes.